src/openforms/registrations/contrib/objects_api/registration_variables.py

```diff
@@ -77,3 +80,69 @@ def get_initial_value(self, submission: Submission | None = None):
         if submission is None:
             return None
         return submission.payments.get_completed_public_order_ids()
+
+
+@register("cosign")
+class Cosign(BaseStaticVariable):
+    name = _("Cosign")
+    data_type = FormVariableDataTypes.object
+
+    def get_initial_value(
+        self, submission: Submission | None = None
+    ) -> FormAuth | None:
+        if not submission or not submission.cosign_complete:
+            return None
+
+        return submission.co_sign_data
+
+
+def get_cosign_value(submission: Submission | None, attribute: AuthAttribute) -> str:
+    if not submission or not submission.cosign_complete:
+
+        return ""
+
+    if submission.co_sign_data["attribute"] == attribute:
+        return submission.co_sign_data["value"]
+
+    return ""
```

Comment:
```suggestion
def get_cosign_value(submission: Submission | None, attribute: AuthAttribute) -> str:
    if not submission or not submission.cosign_complete:
        return ""

    if submission.co_sign_data["attribute"] == attribute:
        return submission.co_sign_data["value"]

    return ""
```
that blank line looks a bit odd :shrug: